src/functions.jl

```diff
@@ -33,12 +29,31 @@ for (mod, funs) in (
     end
 end
 
-# 1 arg before
+# 1 Arg - no default for `dims` keyword
+for (mod, funs) in (
```

Comment:
For code clarity, might be better to change `funs` and `fun` to `funcs` and `func` which are the more common abbreviation of `functions`? Even if it does makes the code less "fun" :)

Reply:
haha 🎊 

For this PR i would prefer to stick with these names as they're already used elsewhere in the file